### ISSUE TITLE
`/etc/os-release` should be included in ShellCheck mocks

### ISSUE THREAD

**Issue by felipecrs** [hadolint/hadolint#1097]
This triggers [SC1091](https://www.shellcheck.net/wiki/SC1091):

```dockerfile
FROM buildpack-deps:noble-curl

SHELl ["/bin/bash", "-euxo", "pipefail", "-c"]

RUN install -m 0755 -d /etc/apt/keyrings; \
  curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc; \
  chmod 0644 /etc/apt/keyrings/docker.asc; \
  echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu $(. /etc/os-release && echo "${UBUNTU_CODENAME}") stable" | tee /etc/apt/sources.list.d/docker.list; \
  apt-get update; \
  apt-get install -y --no-install-recommends docker-ce-cli=5:28.0.0-*; \
  rm -rf /var/lib/apt/lists/*
```

```console
$ hadolint Dockerfile
Dockerfile:5 SC1091 info: Not following: File not included in mock.
```

I believe `/etc/os-release` is such a common case that it should be included in hadolint's mock for shellcheck by default.
